rotkehlchen/db/dbhandler.py

```diff
@@ -3191,7 +3191,11 @@ def ensure_tags_exist(
                 f'{", ".join(unknown_tags)} were found',
             )
 
-    def add_bitcoin_xpub(self, xpub_data: XpubData) -> None:
+    def add_bitcoin_xpub(
+            self,
+            xpub_data: XpubData,
+            blockchain: Literal[SupportedBlockchain.BITCOIN, SupportedBlockchain.BITCOIN_CASH],
+    ) -> None:
```

Comment:
this name and having then the blockchain param was a bit odd for me. Maybe we can rename this function to something that sounds more intuitive?

Reply:
The reason for sticking to bitcoin was because `BCH` is simply a fork of Bitcoin